set.mm

```diff
@@ -101234,10 +101237,9 @@ strictly dominates its argument (or the supremum of all ordinals that
 
   $( The aleph function has at least one fixed point.  Proposition 11.18 of
      [TakeutiZaring] p. 104.  See ~ alephfp for an actual example of a fixed
-     point.  Compare the inequality ~ alephle that holds in general.  Note that
-     if ` x ` is a fixed point, then ` aleph `` aleph `` aleph `` ` ...
-     ` aleph `` x = x ` .  (Contributed by NM, 6-Nov-2004.)  (Revised by Mario
-     Carneiro, 15-May-2015.) $)
+     point.  Compare the inequality ~ alephle that holds in general.
+     (Contributed by NM, 6-Nov-2004.)  (Revised by Mario Carneiro,
```

Comment:
Why did you remove "Note that if ` x ` is a fixed point, then ` aleph `` aleph `` aleph `` ` ... ` aleph `` x = x ` ."?

Reply:
I do not know Norm's intention of this, but probably had one. Since it is not wrong, I would like to keep it.

Reply:
Change reverted.